alinka/widget/containers/main_body/content/settings/settings_tabs/team_member_tab/team_member_table_group.py

```diff
@@ -8,6 +7,98 @@
     QVBoxLayout,
     QWidget,
 )
+from sqlalchemy import delete
+
+from alinka.db.connection import db_session
+from alinka.db.models import TeamMember
+from alinka.db.queries import get_team_members, upsert_team_members
+from alinka.schemas import TeamMemberDbSchema
+
+
+class TeamMemberTableModel(QAbstractTableModel):
+    def __init__(self):
+        self.insert_row = None
+        super().__init__()
+
+    def columnCount(self, parent=QModelIndex()):
+        return len(TeamMember.__table__.columns)
+
+    def data(self, index, role=Qt.DisplayRole):
+        """Return the data for the given index."""
+        if not index.isValid():
+            return None
+        if role == Qt.DisplayRole:
+            if self.insert_row is not None and index.row() + 1 == self.rowCount():
+                return self.insert_row.get(TeamMember.__table__.columns.keys()[index.column()], "")
```

Comment:
Can you split this line to make it more readable?
```suggestion
            if self.insert_row is not None and index.row() + 1 == self.rowCount():
                new_value_column_name = TeamMember.__table__.columns.keys()[index.column()]
                return self.insert_row.get(new_value_column_name, "")
```

Reply:
As I introduced
```python
        self.columns = TeamMember.__table__.columns.keys()
```
in `__init__` this few lines look much more readable. Let's leave them as they are for time being. We will revisit this part, when (I guess) building astract table model (to accomodate both team members and documents tables).

Reply:
I kind of simplified it. Not in the direction, you proposed, but it's much more readable right now.